being-an-adie/roundtables.md

```diff
@@ -0,0 +1,34 @@
+# Roundtables
+
+Roundtables are a core component of the Ada curriculum.  Roundtables meet almost daily and are a place to ask questions, discuss lessons and problems and review content.  The format of a roundtable includes live code sessions with instructors, group activities in the curriculum and question and answer sessions.
+
+Roundtables are a group of 12 students, and the first roundtables will be grouped from your homeroom assignment.  Roundtables will change at the end of each unit (two times total).
+
+Roundtables meet at either 9:30am-10:30am or 10:50-11:50am.
+
+## Logistics
+
+- You can view your roundtable assigment at the [Roundtable assigment page](https://airtable.com/shriFMlEJEU7ZMvqL)__todo CONFIRM__
+- Roundtable instructors facilitating your group may differ day-to-day.
+- Roundtable groups will change the weeks of **April 12th** and **March 17**
```

Comment:
I don't think the March 17th date is correct.

Reply:
Thanks, it should be May 17!